editor/plugins/script_text_editor.cpp

```diff
@@ -254,7 +254,7 @@ void ScriptTextEditor::_set_theme_for_script() {
 	for (const String &doc_comment : doc_comments) {
 		String beg = doc_comment.get_slice(" ", 0);
 		String end = doc_comment.get_slice_count(" ") > 1 ? doc_comment.get_slice(" ", 1) : String();
-		text_edit->add_comment_delimiter(beg, end, end.is_empty());
+		text_edit->add_doc_comment_delimiter(beg, end, end.is_empty());
```

Comment:
Also needs `clear_doc_comment_delimiters()` like in line 238.

Reply:
Good point! I add it! Many thanks!